assets/_sass/components/_keywords.scss

```diff
@@ -0,0 +1,35 @@
+.keywords {
+  ul {
```

Comment:
Take a look at this again. The styles being assigned to the `<ul>` aren't going to be called, because this will compile to: `.keywords ul`. Since the `keywords` class is assigned to the `<ul>`, you want this instead:

```scss
.keywords {
   @extend %tag;
    margin: 0;
    padding: 0;
    padding-bottom: 0.5em;
    text-align: left;
    list-style-type: none;
}
```

Reply:
Fixed! Yes, the `ul` tag was totally ineffective... woops!